File: HW1/main.py
import BMPImg;

def main():
    # mian function must not change
	bmpimg = BMPImg.BMPImg();
	bmpimg.loadPic('img/rabbit.bmp');
	bmpimg.rotate();
	bmpimg.storePic('output/rabbit_ans.bmp');

    bmpimg = BMPImg.BMPImg();
    bmpimg.loadPic('img/flower.bmp');
    bmpimg.PrewittFilter();
    bmpimg.storePic('output/flower_ans.bmp');


if __name__ == '__main__':
	main();

File: HW1/main.cpp
#include <string>
#include "BMPImg.h"

int main(int argc, char* argv[]) {
    // main function must not change
	BMPImg imgrabbit("./img/rabbit.bmp");
	imgrabbit.rotate();
	imgrabbit.storePic("./output/rabbit_ans.bmp");

	BMPImg imgflower("./img/flower.bmp");
	imgflower.PrewittFilter();
	imgflower.storePic("./output/flower_ans.bmp");

	return 0;
}


